hubclient/rapid-scans-client.go

```diff
@@ -15,22 +15,30 @@
 package hubclient
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
 	"github.com/blackducksoftware/hub-client-go/hubapi"
 	log "github.com/sirupsen/logrus"
 	"strconv"
+	"time"
 )
 
 const (
 	apiDeveloperScans     = "/api/developer-scans"
+	apiFullResults        = "/full-result"
 	headerBdMode          = "X-BD-MODE"
 	headerBdDocumentCount = "X-BD-DOCUMENT-COUNT"
 	bdModeAppend          = "append"
 	bdModeFinish          = "finish"
+	OkStatusCode          = 200
```

Comment:
there is a standard `net/http` package that contains these codes already, you might want to use it instead.

https://golang.org/pkg/net/http/#pkg-constants